code/outlierfunction.py

```diff
@@ -0,0 +1,19 @@
+
+# find outliers based on DVARS and FD
+
+def outlier(file_name, save_name):
+    # load data
+    data = np.loadtxt(file_name)
+    outlier = []
+    # set outlier values to 0
+    for i in data:
+        if i <= 0.5:
```

Comment:
is there a reason why we have it set specifically to 0.5? just thinking of writing test functions and this would not be too comprehensive if you have other types of file/data like variance or mean